core/core_bind.cpp

```diff
@@ -572,7 +572,7 @@ void _OS::print_all_textures_by_size() {
 		ResourceCache::get_cached_resources(&rsrc);
 
 		for (List<Ref<Resource>>::Element *E = rsrc.front(); E; E = E->next()) {
-			if (!E->get()->is_class("ImageTexture")) {
+			if (!E->get()->is_class("Texture")) {
```

Comment:
I don't know if there was a reason to limit to `ImageTexture`, but it seems much more common to me to have `StreamTexture`s in use, this handles both.